CMakeLists.txt

```diff
@@ -312,6 +312,8 @@ endif()
 # dependencies.
 add_library(umf_headers INTERFACE)
 
+set(CMAKE_INSTALL_RPATH "\$ORIGIN")
```

Comment:
please move this to "install(FILES .. " with other installation-related code

Reply:
Can't move it there as this variable has to be set before creating a umf target. Moved it to the CMakeLists file right before defining of a umf target.